subsys/suit/stream/Kconfig

```diff
@@ -37,7 +37,6 @@ config SUIT_STREAM_SINK_RAM
 
 config SUIT_STREAM_SINK_EXTMEM
 	bool "Enable external memory sink"
-	depends on SSF_EXTMEM_SERVICE_ENABLED
```

Comment:
Does the code is prepared to work without the service or does this dependency is expressed in a different way?

Reply:
The config that should be in this statement is only defined in sdfw and is not accessible in sdk-nrf.